README.md

```diff
@@ -94,54 +94,48 @@ For RabbitMQ
 docker pull rabbitmq:3-management
 docker run --name r-mailing -it --hostname my-rabbit -p 15672:15672 -p 5672:5672 rabbitmq:3-management
 ```
+### Creating Database Tables:
 
-### Add new user
-
-- To be able to log in add new Employee with your Retro Rabbit email to the TestData.cs
-- Add new Migration for your user
-
-```powershell
-add-migration newMigrationYouCanChooseName
-```
-
-### Run migration
-
-Open Visual Studio 2022 and open the RGO-Server project file. Pull up the nuget package manager console:
-**_Tools_** -> **_NuGet Package Manager_** -> **_Package Manager Console_**
-Make sure the **Default project** is **_RGO.UnitOfWork_**.
-
----
-
-![Image of Package Manager Console](./RGO-UnitOfWork-example.png)
-
+1. Open Visual Studio and go to the package manager.
+2. Change the default project to RR.UnitOfWork.
+3. Run the following commands:
+   
 ```powershell
-Update-Database
+add-migration migrationName
+update-database
 ```
 
-### Checking new user added to the DB you made
-
-- Install **PgAdmin** beforehand. If you locally installed **_PostgreSQL_** be warned that it may interfear with your attempts to connect to the database(Docker).
-- https://www.pgadmin.org/download/pgadmin-4-windows/
+Congratulations! You have now successfully created a database with tables.
 
----
+### Populating Database with Dummy Data:
 
-- Register new RGO server

```

Comment:
Add the Register RGO server stuff back as well, new users or users with no programming experience needs to know how to set this up